src/RNCamera.js

```diff
@@ -461,7 +468,7 @@ export default class Camera extends React.Component<PropsType, StateType> {
     }
   }
 
-  _convertNativeProps(props: PropsType) {
+  _convertNativeProps({ children, ...props }: PropsType) {
```

Comment:
@sibelius I excluded `children` from the props here? Will this result in a performance improvement? IDK if the children would be passed to the native instance as a string or something like that

Reply:
Not sure either

Reply:
I mean it wont hurt not passing it because it is not intended to be passed down 😉